artifactory/commands/transferconfig/utils.go

```diff
@@ -37,6 +38,23 @@ func archiveConfig(exportPath string, configXml string) (buffer *bytes.Buffer, r
 		}
 	}()
 
+	// In some versions of Artifactory, the file access.bootstrap.json has a typo in its name: access.boostrap.json.
+	// If this is the case, rename the file to the right name.
+	accessBootstrapPath := filepath.Join(exportPath, "etc", "access.bootstrap.json")
+	accessBootstrapExists, err := fileutils.IsFileExists(accessBootstrapPath, false)
+	if err != nil {
+		return nil, err
+	}
+	if !accessBootstrapExists {
+		err = fileutils.MoveFile(filepath.Join(exportPath, "etc", "access.boostrap.json"), accessBootstrapPath)
+		if err != nil {
+			if os.IsNotExist(err) {
+				return nil, errorutils.CheckErrorf("%s: the file was not found or is not accessible", accessBootstrapPath)
+			}
+			return nil, err
+		}
+	}
+
```

Comment:
I think this logic would look better in a new function.
This new function can have a nice unit test.